supervisor/shared/autoreload.c

```diff
@@ -69,6 +71,9 @@ inline bool autoreload_is_enabled() {
 }
 
 void autoreload_start() {
+    if (autoreload_delay_ms == 0) {
```

Comment:
This is some kind of "on the very first run after hard reset" check?

Reply:
Comment added.